sopel/modules/reload.py

```diff
@@ -33,54 +32,40 @@ def f_reload(bot, trigger):
             'medium': collections.defaultdict(list),
             'low': collections.defaultdict(list)
         }
+        bot.shutdown_methods.clear()
+        bot.scheduler.clear_jobs()
```

Comment:
With the refactor in `bot.py` to remove only the specified job, why is the `reload` module now clearing the whole queue? Why is `reload` clearing all shutdown methods for the entire bot instead of only the specified module? Keeping #831 in mind, this refactor is a good opportunity to stop clearing things that shouldn't be.